pyrdp/ui/PlayPauseButton.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, parent: QWidget = None, playText = "Play", pauseText = "Pause
         self.pauseText = pauseText
         self.playing = False
         self.setPlaying(self.playing)
+        self.setFixedWidth(100)
```

Comment:
Why this change?

Reply:
This gets a constant fixed width to the play/pause button as, before this, when the label toggled from `Play` to `Pause` the width changed. It is more of a design change.